runner/watcher.go

```diff
@@ -37,12 +38,24 @@ func watchFolder(path string) {
 
 func watch() {
 	root := root()
+	ex := excludeDir()
+
 	filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
 		if info.IsDir() && !isTmpDir(path) {
 			if len(path) > 1 && strings.HasPrefix(filepath.Base(path), ".") {
 				return filepath.SkipDir
 			}
 
+			if len(ex) > 0 {
+				for _, ep := range strings.Split(ex, ",") {
```

Comment:
Comma is a valid character in path on some operating systems (Windows for example)